script/create-dist

```diff
@@ -110,9 +116,15 @@ INCLUDE_DIRS = [
   'build',
   'cc',
   'chrome/browser/ui/libgtk2ui',
+  'components/cdm',
   'components/devtools_discovery',
   'components/devtools_http_handler',
   'components/os_crypt',
+  'components/component_updater',
+  'components/content_settings',
+  'components/nacl',
+  'components/omnibox',
```

Comment:
You're right I don't know if they are required. I will try to compile electron without these dependencies and tell you if we need it or not